import-service/functions/importFileParser.ts

```diff
@@ -0,0 +1,40 @@
+import { S3Handler } from "aws-lambda";
+import { corsWrapper } from "../utils";
+import csv from 'csv-parser';
+
+export const importFileParser = (bucket): S3Handler => {
+    return async (event, _context) => {
+        try {
+            for (const record of event.Records) {
+                const params = {
+                    Bucket: process.env.BUCKET_NAME,
+                    Key: record.s3.object.key
+                }
+                const s3Stream = bucket.getObject(params).createReadStream();
+                await new Promise((resolve, reject) => {
+                    s3Stream.pipe(csv())
+                        .on('data', (data) => console.log(data))
+                        .on('error', (error) => reject(error))
+                        .on('end', () => {
+                            resolve();
+                        });
+                });
+                await bucket.copyObject({
+                    Bucket: process.env.BUCKET_NAME,
+                    CopySource: process.env.BUCKET_NAME + '/' + record.s3.object.key,
+                    Key: record.s3.object.key.replace('uploaded', 'parsed')
+                }).promise();
+                await bucket.deleteObject({
+                    Bucket: process.env.BUCKET_NAME,
+                    Key: record.s3.object.key
+                }).promise();
+            }
+            return corsWrapper({
+                statusCode: 200,
```

Comment:
As I understand the correct status code should be `202` because this lambda triggered by `event`.

Reply:
Probably there should not be response at all.

Reply:
Well, I can agree with @asamurai, but it was a part of a task, so I just returned this default response. At the same time, I don't think that 202 is appropriate status code, because it tells us, that request was accepted, but it was not processed, and here we have all logic and we don't delegate anything to other services, so I think that 202 is not a good decision(@AndreyS55)